src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Input validation purpose
+ */
+
+public class Contact {
+
+    public final String value;
+    public boolean isPrivate;
+
+    public Contact(String _value, boolean _isPrivate) {
+        value = _value;
+        isPrivate = _isPrivate;
+    }
+
+    @Override
+    public int hashCode()
```

Comment:
In general, `equals()` and `hashCode()` methods should be overridden together. Find out why!